examples/smpi/replay/replay.cpp

```diff
@@ -4,7 +4,13 @@
  * under the terms of the license (GNU LGPL) which comes with this package. */
 
 #include "xbt/replay.hpp"
+#include "simgrid/s4u.hpp"
```

Comment:
If you can reduce the amount of includes to the exact ones you need in S4U, this will speed up rebuilds.

Reply:
``simgrid/s4u/Actor.hpp`` is now included instead.